expressEngine/lib/service/db.js

```diff
@@ -27,6 +27,22 @@ class DBOperation {
     }
   }
 
+  updateOne = async (collection, dataToUpdate, filter) => {
+    try {
+      return await collection.updateOne(filter, { $set: dataToUpdate });
+    } catch (err) {
+      return { error: err.message };
+    }
+  }
+
+  updateMany = async (collection, dataToUpdate, filter) => {
```

Comment:
In case the user wants to update 'unitPrice' of all products of a certain category('name' is same of products). then is it possible to do so?

Reply:
yes

Reply:
you have not used 'updateMany' anywhere. then how is it possible?

Reply:
I have Created UpdateMany Operation in database for using it if needed in any module,
The file changes you are seeing in this file is because I created this branch not from develop but from feature/115

Reply:
till now, considering your previous branches. Have you made that functionality where i can do so?

Reply:
no such issue is created till now for that functionality in the product